packages/code-studio/src/log/LogExport.ts

```diff
@@ -84,6 +86,7 @@ export async function exportLogs(
   const folder = zip.folder(fileNamePrefix) as JSZip;
   folder.file('console.txt', logHistory.getFormattedHistory());
   folder.file('redux.json', getReduxDataString());
+  folder.file('metadata', JSON.stringify(metadata));
```

Comment:
```suggestion
  folder.file('metadata.json', JSON.stringify(metadata));
```

Reply:
I made that change before creating the PR, but I guess didn't push the commit